### ISSUE TITLE
Rails Importmap - Version 1.6.0 / 1.6.1 not found

### ISSUE THREAD

**Issue by Laylatame** [themesberg/flowbite#398]
**Describe the bug**
I installed Flowbite V1.6.1 in my Rails project (Rails 7.0.2). I followed all instructions on [Flowbite Get Started for Rails](https://flowbite.com/docs/getting-started/rails/) However, when I load my application I get a message in the console saying `"DevTools failed to load source map: Could not load content for https://unpkg.com/flowbite@1.6.0/dist/flowbite.turbo.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE"`.

Basically, it can't find the file for importmap for version 1.6.0. I tried loading an older version (1.5.5) and it loads just fine, but functionality for certain items is lost.

**Expected behavior**
I'm trying to use importmap to allow for Flowbite to work after turbo stream responses, which I haven't been able to do.

**Comment by zoltanszogyenyi** [comment]
Hello @Laylatame,

It looks like there are some problems with with UNPKG - I've tried the links from CDNJS and they seem to work.

Can you please try with:

```
https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.1/flowbite.turbo.min.js
```

Let me know if it works.

Cheers,
Zoltan

**Comment by Laylatame** [comment]
Hi @zoltanszogyenyi, I tried it and it's still not working.
<img width="677" alt="Screen Shot 2023-01-11 at 9 51 20" src="https://user-images.githubusercontent.com/5714517/211852474-79c94455-962b-4bbf-bb87-6872f228abd7.png">

**Comment by zoltanszogyenyi** [comment]
Hey @Laylatame,

It seems that the JS mapping files are missing so we'll build them inside the `dist/` folder for the 1.6.2 version.

This error shouldn't affect the functionality of the application, though. These `.js.map` files are used to debug the code in your browser.

**Comment by zoltanszogyenyi** [comment]
Hey @Laylatame,

Update: indeed there was a type on our side and instead of moving the `flowbite.turbo.min.js.map` we moved the `flowbite.turbo.map.js` file which din not exist - so the file will be available for today's `v1.6.2` release. Stay tuned!

**Comment by Laylatame** [comment]
Hi @zoltanszogyenyi,

Sorry for the late response, just got to it but it is still not working. I updated flowbite to use V1.6.2 and added the new cdn file to my importmap and I am still getting the same error message.
<img width="249" alt="Screen Shot 2023-01-13 at 12 59 33" src="https://user-images.githubusercontent.com/5714517/212398005-751c8e1b-0d83-451c-99b0-941158a8d5e9.png">

**Comment by apricot13** [comment]
Just adding that the current version `1.6.5` is giving a 404 too from the documentation:  https://flowbite.com/docs/getting-started/rails/#importmap

https://cdnjs.cloudflare.com/ajax/libs/flowbite/1.6.5/flowbite.turbo.min.js
